source/client-dependencies.txt

```diff
@@ -1,14 +1,258 @@
+
+         .. code-block:: yaml
+
+            security:
+              enabled: true
+
+         .. note::
+
+            If you want to omit adding ``mechanism`` and ``source`` 
+            to the username, add and set these option/value pairs 
+            to your :doc:`mongosqld </reference/mongosqld>` 
+            configuration file:
+   
+            - :setting:`security.defaultMechanism` ``: PLAIN``
+            - :setting:`security.defaultSource` ``: $external``
+   
+            The resulting :ref:`security <msqld-conf-security-options>`
+            block in your :doc:`mongosqld </reference/mongosqld>` 
+            configuration file should look like this:
+   
+            .. code-block:: yaml
+   
+               security:
+                 enabled: true
+                 defaultMechanism: "PLAIN"
+                 defaultSource: "$external"
+
+            For more information on the |bi-short|'s configuration file settings,
+            see:
+
+            - :setting:`security.enabled`
+            - :setting:`security.defaultMechanism`
+            - :setting:`security.defaultSource`
+
+     - id: kerberos
+       content: |
+
+         To enable Kerberos (``GSSAPI``) as your 
+         |bi-short| authentication mechanism, you must add 
+         :setting:`security.enabled` ``: true`` to your 
+         :doc:`mongosqld </reference/mongosqld>` configuration file:
+
+         .. code-block:: yaml
+
+            security:
+              enabled: true
+
+         .. note::
+
+            If you want to omit adding ``mechanism`` and ``source`` 
+            to the username, add and set these option/value pairs 
+            to your :doc:`mongosqld </reference/mongosqld>` 
+            configuration file:
+   
+            - :setting:`security.defaultMechanism` ``: GSSAPI``
+            - :setting:`security.defaultSource` ``: $external``
+   
+            The resulting :ref:`security <msqld-conf-security-options>`
+            block in your :doc:`mongosqld </reference/mongosqld>` 
+            configuration file should look like this:
+   
+            .. code-block:: yaml
+
+               security:
+                 enabled: true
+                 defaultMechanism: "GSSAPI"
+                 defaultSource: "$external"
+                 gssapi:
+                   hostname: "<yourHostname>"
+                   serviceName: "mongosql"
+
+         For more information on the |bi-short|'s configuration file settings,
+         see:
+
+         - :setting:`security.enabled`
+         - :setting:`security.defaultMechanism`
+         - :setting:`security.defaultSource`
+         - :setting:`security.gssapi.hostname`
+         - :setting:`security.gssapi.serviceName`
+
+         For more information, see :doc:`/tutorial/kerberos`.
+
+Client Authentication
+---------------------
+
+If your :doc:`BI tool <client-applications>` is using the
 :doc:`MongoDB ODBC Driver </reference/odbc-driver>`, the driver will
```

Comment:
As long as you've got this file open, I recommend not capitalizing "Tool" in "BI Tool" lower down in this paragraph.